codalab/apps/web/static/js/worksheet/table_item_interface.jsx

```diff
@@ -89,6 +89,7 @@ var TableItem = React.createClass({
                         <thead>
                             <tr>
                                 {header_html}
+                                <th className='table-column-_dropdown'></th>
```

Comment:
Why is there an underscore before "dropdown"? If no particular reason, can we rename to `table-column-dropdown`?